app/controllers/app/purchases_controller.rb

```diff
@@ -1,5 +1,15 @@
 class App::PurchasesController < App::BaseController
   def index
-    @purchases = current_user.purchases.order(created_at: :desc).includes(:item)
+    @purchases = current_user.purchases.order(created_at: :desc).includes([:item, :delivery_company])
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Layout/LineLength: Line is too long. [101/100]